uiowa/uiowa
# Feature: Updated headline underline style

# How to test

```
rm -rf ./node_modules
yarn cache clean
yarn install
yarn workspace uids_base gulp --development
blt ds --site=sandbox.uiowa.edu
drush @sandbox.local uli
```

- Compare https://sandbox.prod.drupal.uiowa.edu/headline-underline with https://sandbox.local.drupal.uiowa.edu/headline-underline
- Test https://sandbox.local.drupal.uiowa.edu/kitchen-sink by changing the text areas to a different background color and verify that the underline is visible.  Example, a gold background section with a text area with a gray background.
- Look at any other sites that you can think of using this headline style


I've added a number of screenshots in details below with examples of how this will look on sites:

<details>
<img width="1414" alt="Screen Shot 2021-11-05 at 4 38 13 PM" src="https://user-images.githubusercontent.com/1036433/140978929-ce683cb1-8a9f-4b20-8d8b-f0da23b79220.png">
<img width="1875" alt="Screen Shot 2021-11-05 at 4 37 44 PM" src="https://user-images.githubusercontent.com/1036433/140978933-ef29a143-badf-4c51-9103-561ed7adad10.png">
<img width="1627" alt="Screen Shot 2021-11-05 at 4 34 03 PM" src="https://user-images.githubusercontent.com/1036433/140978935-8dc3fab9-2318-452a-80a8-911785e4afda.png">
<img width="1874" alt="Screen Shot 2021-11-05 at 4 30 37 PM" src="https://user-images.githubusercontent.com/1036433/140978807-bef9e062-5d0e-45dd-aa16-0495ff9b5762.png">
<img width="1766" alt="Screen Shot 2021-11-05 at 4 31 47 PM" src="https://user-images.githubusercontent.com/1036433/140978826-364b0d13-b032-43a4-b408-0b0a7e63eccd.png">
<img width="1808" alt="Screen Shot 2021-11-05 at 4 32 23 PM" src="https://user-images.githubusercontent.com/1036433/140978836-34fdc7b8-f93d-41f8-8702-5aa8cef8275d.png">
<img width="1398" alt="Screen Shot 2021-11-05 at 4 37 35 PM" src="https://user-images.githubusercontent.com/1036433/140978843-991ec5d3-b52d-4659-b3e9-158b0c38931f.png">
</details>
